src/WebJobs.Extensions.DocumentDB/DocumentDBAttribute.cs

```diff
@@ -90,5 +91,13 @@ public DocumentDBAttribute(string databaseName, string collectionName)
         /// collection.
         /// </summary>
         public int CollectionThroughput { get; set; }
+
+        /// <summary>
+        /// Optional.
+        /// When specified on an input binding using an <see cref="IEnumerable{T}"/>, defines the query to run against the collection. 
```

Comment:
What happens if someone specifies a SqlQuery that returns a single row, and binds it to a singleton type? Do we somehow prevent this?